C/Guessing Game/main.c

```diff
@@ -0,0 +1,43 @@
+#include <stdio.h>

```

Comment:
- Remove the exe file
- add comments to this program
- add GIF demonstrating the game
- place that GIF in the same folder Guessing Game  you created
- edit readme of C folder with your entry